recipes/ostaphep/activate.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+export OSTAPDIR="${CONDA_PREFIX}"
```

Comment:
Is this variable needed for anything? Or can the activation scripts be removed?

(It’s not a problem but it’s nicer to not have them if you can.)